### PR TITLE
Sections and severity

### PR DESCRIPTION
**Pull request by sboisvert**
Add some logical separation of stuff and do a first pass at severity

### CODE REVIEW

File: WordPress-VIP-Go/ruleset.xml [hunk 1]
```diff
@@ -5,79 +5,171 @@
...
+
+	<rule ref="WordPress.WP.PreparedSQL" >
+		<severity>6</severity>
+	</rule>
+
+
+	<!-- VIP Uncached remote calls usually severity 4 -->
+	<rule ref="WordPressVIPMinimum.VIP.RestrictedFunctions.file_get_contents">
+		<severity>4</severity>
+		<message>%s() is uncached. If this is being used to query a remote file please use wpcom_vip_file_get_contents() instead. If this is</message>
```
Comment (vaurdan) [inline WordPress-VIP-Go/ruleset.xml]: I believe this message is incomplete.

File: WordPress-VIP-Go/ruleset.xml [hunk 2]
```diff
@@ -5,79 +5,171 @@
...
+		<severity>4</severity>
+		<message>https://vip.wordpress.com/documentation/uncached-functions/ talk about adding the parameters</message>
+	</rule>
+	<rule ref="WordPressVIPMinimum.VIP.RestrictedFunctions.get_posts_get_posts">
+		<type>warning</type>
+		<severity>4</severity>
+		<message>https://vip.wordpress.com/documentation/uncached-functions/ talk about adding the parameters</message>
+	</rule>
+
+	<!-- Why is this here at level 1 is it not the same as the previous? -->
```
Comment (vaurdan) [inline WordPress-VIP-Go/ruleset.xml]: I think this includes all the functions [inside the `get_posts` sniff](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Sniffs/VIP/RestrictedFunctionsSniff.php#L163-L171).

Comment (sboisvert) [reply WordPress-VIP-Go/ruleset.xml]: Do we want that?

**Comment by tomjn** [comment]
Since we're doing things with standard review now, this PR is ultra high priority.

I would recommend that even if the process of severity values is incomplete that it be merged ASAP with follow up PRs. Holding this up to reach a final completion is only going to cause issues

**Pull request closed without merging**